README.md

```diff
@@ -8,32 +8,56 @@ An updated beacon supporting [ GA4GH API 1.0 ][ga4gh_api1]
 This README only gives a brief overview of the tool, for a more complete reference, please check out our docs: www.clinicalgenomics.se/cgbeacon2
 
 Table of Contents:
-1. [ Prerequisites ](#prerequisites)
-2. [ Installation ](#installation)
+1. [ Running the app using Docker ](#docker)
+2. [ Installation using pip in a virtual environment](#installation)
 3. [ Loading demo data into the database ](#Loading)
 4. [ Server endpoints ](#endpoints)
     - [ Obtaining beacon info ](#info)
     - [ Running queries ](#query)
 5. [ Web interface ](#webform)
 
-<a name="prerequisites"></a>
-## Prerequisites
-- Python 3.6+
-- A working instance of **MongoDB**. From the mongo shell you can create a database using this syntax:
+
+<a name="docker"></a>
+## Installing and running the app using Docker
```

Comment:
It would be nice if there was some test data and instruction for how to populate the database for testing only

Reply:
Good idea, I'll add the command to load the demo data!

Reply:
The command to load the data is actually already included in the demo, but I'll comment about it in the readme and the docs